### PR TITLE
Viewpoint follow object rotation doc

### PR DESCRIPTION
**Pull request by DavidMansolino**
linked with omichel/webots #3811

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

File: reference/viewpoint.md [hunk 1]
```diff
@@ -39,11 +40,13 @@ The `near` and the `fieldOfView` fields define together the viewing frustum. Any
 (standing between the camera and the near plane) won't appear.
 
 The `follow` field can be used to specify the name of a robot (or other object)
-that the viewpoint needs to follow during the simulation. If the string is
+that the viewpoint will follow in translation (travelling movement) during the simulation. If the string is
```
Comment (stefaniapedrazzi) [inline reference/viewpoint.md]: It seems that travelling with two ``l`` is used especially in British English whether in American English the version with a single ``l`` is more usual.

Comment (omichel) [inline reference/viewpoint.md]: Ok, thanks for pointing that out. I will fix it.

**Comment by omichel** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:+1:    (added documentation about the new menu).

**Comment by omichel** [comment]
:+1:

**Pull request merged**